roles/docker/tasks/systemcontainer_crio.yml

```diff
@@ -3,6 +3,9 @@
 - set_fact:
     l_insecure_crio_registries: "{{ '\"{}\"'.format('\", \"'.join(openshift.docker.insecure_registries)) }}"
   when: openshift.docker.insecure_registries
+- set_fact:
+    l_additional_crio_registries: "{{ '\"{}\"'.format('\", \"'.join(openshift.docker.additional_registries)) }}"
+  when: openshift.docker.additional_registries
```

Comment:
We need to specify "docker.io" here as well. Cri-o is not gonna default to docker like projectatomic/docker does ;)